indico/core/signals/__init__.py

```diff
@@ -14,5 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
-from indico.core.signals import acl, agreements, attachments, category, event, event_management, plugin, users
+from indico.core.signals import (acl, agreements, attachments, category, event, event_management, menu, plugin, users)
 from indico.core.signals.core import *
+
+__all__ = ('acl', 'agreements', 'attachments', 'category', 'event', 'event_management', 'menu', 'plugin', 'users', 'rb')
```

Comment:
at least here I think `__all__` is not a good idea - if you add a new signal e.g. to signals.core.py chances are good that you don't look at this file (and repeating all the core signals here feels ugly in any case)

actually all the core signals are missing in `__all__` here right now